packages/api/src/useFetch.ts

```diff
@@ -18,7 +18,7 @@ const useFetch = <T extends TSocketEndpointNames>(name: T, ...props: TSocketAcce
 
     return useQuery<TSocketResponseData<T>, unknown>(getQueryKeys(name, payload), () => send(name, payload), {
         ...options,
-        enabled: options?.enabled ?? is_authorized,
+        enabled: options?.enabled && is_authorized,
```

Comment:
@farzin-deriv ?

Reply:
@mahdiyeh-deriv Had a talk with @sergei-deriv, It should be `&&` as he explained to me 🙇🏻